module/PowerNSX.psm1

```diff
@@ -13969,6 +14001,174 @@ function Disable-NsxEdgeSsh {
 
 }
 
+function Get-NsxcliSettings {
```

Comment:
No need for this cmdlet if making part of set-nsxedge...  If it is to be a seperate cmdlet though, the name should have the word edge in it!